plugin/storage/cassandra/factory.go

```diff
@@ -38,13 +44,14 @@ type Factory struct {
 
 	primaryConfig  config.SessionBuilder
 	primarySession cassandra.Session
-	// archiveSession cassandra.Session TODO
+	archiveConfig  config.SessionBuilder
```

Comment:
I feel that the `archiveConfig` can be set to a higher default consistency level than the `primaryConfig`. What do you think?

Reply:
it's fully configurable by user. Given how marginal this feature is overall, I don't think introducing custom defaults per storage class is worth the complexity in the code.